portfolio/src/main/webapp/index.html

```diff
@@ -7,31 +7,71 @@
     <script src="script.js"></script>
   </head>
   <body>
+
+    <!-- Sticky Navbar -->
     <div id="navbar">
         <div id="navbar-title">
             Mafe's Portfolio
         </div>
         <div id="navbar-right">
-            <a href="#aboutme">About Me</a>
-            <a href="#link2">Link 2</a>
-            <a href="#link3">Link 3</a>
+            <a href="#void">About Me</a>
+            <a href="#void">Projects</a>
+            <a href="#void">Work Experience</a>
+            <a href="#void">Contact Me</a>
         </div>
     </div>
+
+    <!-- Site Content -->
     <div id="content">
-      <h1>About Me</h1>
-      <!-- Introductoy Paragraphy -->
-      <p>Hello! My name is Mafe and I am currently a STEP intern at Google. 
+
+      <!-- About Me Section -->
+      <div class="section" id="aboutme">
+          <h1>About Me</h1>
+          <p>Hello! My name is Mafe and I am currently a STEP intern at Google. 
           I was born and raised in Lima, Peru and moved to the US about 6 years ago to pursue my college degree. 
           It wasn't until after I graduated from college that I discovered my passion for Computer Science and programming, 
           so I decided to go back to school to complete a second bachelors in Computer Science. 
           I just wrapped up my first year as a CS student and I am loving every minute of it!
-      <br><br>
-      Here's a picture of my dog Arlo:
-      <br>
-      <img src="/images/arlo.jpg" alt="My dog Arlo"></p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+          </p>
+          <p>In my free time I enjoy cooking, playing tennis, skiing or snowboarding, and hiking with my dog Arlo. 
+          Arlo is a 4-year-old English Setter who enjoys eating, sleeping, eating, hiking, and sleeping.
+          </p>
+          
+          <!-- Arlo Sub-Section -->
+          <center>
+              <img src="/images/arlo.jpg" alt="My dog Arlo">
+              <p><i>Click here to make Arlo talk:</i></p>
+              <button onclick="addRandomArloMessage()">Talk</button>
+              <div id="arlo-talks-container"></div>
+              </center>
+      </div>
+
+      <!-- Projects Section -->
+      <div class="section" id="projects">
+          <h1>Projects</h1>
+          <ul>
+              <li><b>Finance:</b> A mock stock-trading website implemented using Python, SQL, and Flask, where a user can register, login, change passwords, buy and sell stocks, and see their transaction history.</li>
+              <li><b>Liberty Slot Machine:</b> A three-part program implemented in Python to simulate a game of the Liberty Bell Slot Machine.
+              <ol>
+                  <li>Tests a serious of known spins and outputs expected payout.</li>
```

Comment:
> serious

series?

Reply:
Typo, will fix, thanks for catching that!